README.md

```diff
@@ -40,15 +40,26 @@ There are several options how you can run examples:
 * from your IDE
 * from a command line
 
+## Configure insighredge/bin/setenv-overide
+ 
+###### Linux
+    update /insightedge/bin/setenv-overide.sh    
+     export SPARK_LOCAL_IP=127.0.0.1
+##### Windows
+    update /insightedge/bin/setenv-overide.bat
+     set SPARK_LOCAL_IP=127.0.0.1
+
 #### Starting local environment
 
+
+
 Prior to executing example application, you have to start the Data Grid and deploy an empty space on it. You can do it using `demo` mode:
 ```bash
 # Linux:
-./insightedge/bin/insightedge --mode demo
+./insightedge/bin/insightedge demo
 
 # Windows:
-insightedge\sbin\insightedge.cmd --mode demo
+insightedge\sbin\insightedge.cmd demo
```

Comment:
sbin is dead, and we're using bat and not cmd for the new script:
insightedge\sbin\insightedge.cmd => insightedge\bin\insightedge.bat